docs/en/05_Contributing/01_Code.md

```diff
@@ -86,6 +86,20 @@ If you are changing existing APIs, introducing new APIs or major new features,
 please review our guidance on [supported versions](release_process#supported-versions).
 You will need to change the branch for your pull request accordingly.
 
+SilverStripe public APIs explicitly include:
+ - namespaces, classes, interfaces and traits
+ - public and protected scope (including methods, properties and constants)
+ - global functions, variables
+ - yml configuration file structure
```

Comment:
As long as this also includes `yml` variable typing. You shouldn't be able to change an `int` config setting to `int|string` in a patch release, for example.

Reply:
fixed